src/components/SvgIcon/SvgIcon.stories.tsx

```diff
@@ -1,6 +1,6 @@
-import * as ColorTokens from "@chanzuckerberg/eds-tokens/lib/ts/colors";
 import type { StoryObj } from "@storybook/react";
 import React from "react";
+import * as ColorTokens from "../../../lib/tokens/ts/colors";
```

Comment:
Not totally sure if it's worth it, but we could create a path alias to turn this relative import into something like

```ts
import * as ColorTokens from "lib/tokens/ts/colors";
```

Reply:
mm yeah. maybe if these stick around 😁